lib/tramway/decorators/association.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Tramway
+  module Decorators
+    # Module for defining association instance-level methods
+    module Association
+      private
+
+      # :reek:UtilityFunction { enabled: false }
+      def decorate_has_many_association(assoc)
+        AssocDecoratorHelper.decorate_has_many_association(assoc)
+      end
+    end
+
+    # Module for defining association class-level methods
+    #
+    module AssociationClassMethods
+      def associations(*associations)
+        associations.each do |assoc|
+          association assoc
+        end
+      end
+
+      # has_and_belongs_to_many is not supported for now
```

Comment:
why not? Is there any difference from `has_many`?

Reply:
I don't know the differences exactly, but I don't have good enough examples to test it.

Reply:
why, you have tests :) If not, it should be not just a comment, but FIXME or TODO, and be mentioned in readme that feature is not working with has_and_belongs_to_many
(though I think it could work event without changes - but it should be tested)